src/script/conversation/EventMapper.ts

```diff
@@ -188,7 +189,7 @@ export class EventMapper {
     }
 
     if (event.reactions) {
-      originalEntity.reactions(event.reactions);
+      originalEntity.reactions(userReactionMapToReactionMap(event.reactions));
```

Comment:
mapping the reactions to the new format (making sure the old format is never lying around in the app state)